### ISSUE TITLE
How to increase the font size in "TPJWinMsgDlg"

### ISSUE THREAD

**Issue by luapfr** [ddablib/msgdlg#3]
How to increase the font size in "TPJWinMsgDlg" you could create a property to increase the font size would be very good.

**Comment by delphidabbler** [comment]
First thought is that this might be difficult because the component is a thin wrapper round the Windows API, so I'm not sure if I can get at the front.

I'll have a look though to see if anything can be done.

**Comment by luapfr** [comment]
It would be very good, because I looked on the internet and there is nothing I can do to increase the font size, if your colleague could do that it would be great.

**Comment by delphidabbler** [comment]
I've reviewed the code and done a bit of research online and sadly I can't find a way of altering the font size. All the threads I've seen suggest altering font scaling via Windows personalisation, which doesn't help!

It might be possible to hack the font size in TPJVCLMsgBox though. If that would be any use to you I'll have a look at how to do it.

It's ten years since I updated this code, so I've forgotten how it works!!
